fml/BUILD.gn

```diff
@@ -5,6 +5,11 @@
 import("//build/fuchsia/sdk.gni")
 import("$flutter_root/testing/testing.gni")
 
+if (is_fuchsia) {
+  import("$flutter_root/tools/fuchsia/common_libs.gni")
```

Comment:
nit: It might be nice to have a `fuchsia_test_archive` template, that specifies the `common_libs` and `testonly = true` by default.